packages/azure-monitoring/eslint.config.mjs

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import hmppsConfig from '@ministryofjustice/eslint-config-hmpps'
+
+export default [
+  ...hmppsConfig(),
+  { ignores: ['src/sample/**.ts'] },
```

Comment:
Couldn't use the `extraIgnorePaths` option as got an error around:
`You are linting ".", but all of the files matching the glob pattern "." are ignored`